bundle/manifests/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -527,11 +527,11 @@ spec:
   minKubeVersion: 1.19.0
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.60.0
+  replaces: jaeger-operator.v1.61.0
```

Comment:
Its 1.60.1 right?

Reply:
Good catch. I'm not sure why the automation did this